lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -1014,6 +1015,7 @@
     <sniffer type="galaxy.datatypes.tabular:CSV"/>
     <sniffer type="galaxy.datatypes.tabular:TSV"/>
     <sniffer type="galaxy.datatypes.tabular:MatrixMarket"/>
+    <sniffer type="galaxy.datatypes.tabular:CMAP"/>
```

Comment:
I think the sniffer for CMAP needs to appear prior to CSV and TSV (and, possibly, ConnectivityTable as well), as the format for cmap is more rigid (see docs: https://docs.galaxyproject.org/en/master/dev/data_types.html#step-2-sniffer - step 2). If you agree, please move it to line 1014, right before ConnectivityTable.

Reply:
I moved it to line 1014.